app/src/full/java/org/openobservatory/ooniprobe/common/FirebaseMessagingService.java

```diff
@@ -25,13 +24,6 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
 			try {
 				//TODO-FUTURE we can use click_action instead of type
 				JSONObject data = new JSONObject(params.toString());
-				if (data.getString("type").equals("open_href")) {
-					Intent intent = new Intent(getApplicationContext(), NotificationDialogActivity.class);
-					intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-					intent.putExtra("message", remoteMessage.getNotification().getBody());
-					intent.putExtra("payload", data.getString("payload"));
-					getApplicationContext().startActivity(intent);
-				}
```

Comment:
Here we can possibly remove the whole method, right?

Reply:
This will be done in the countly branch, it's not the scope of this branch to remove firebase

Reply:
Please, add a TODO comment mentioning that, then?

Reply:
It has already been DONE, in the other branch.